tests/conftest.py

```diff
@@ -1814,3 +1815,10 @@ async def _async_call(
 def snapshot(snapshot: SnapshotAssertion) -> SnapshotAssertion:
     """Return snapshot assertion fixture with the Home Assistant extension."""
     return snapshot.use_extension(HomeAssistantSnapshotExtension)
+
+
+@pytest.fixture
+def disable_block_async_io_after() -> Generator[Any, Any, None]:
```

Comment:
either `disable_block_async_io_after_test` or `disable_block_async_io`

Reply:
renamed to `disable_block_async_io`